src/ESLintLinter.php

```diff
@@ -160,6 +160,13 @@ protected function parseLinterOutput($path, $err, $stdout, $stderr) {
 
     foreach ($json as $file) {
       foreach ($file['messages'] as $offense) {
+        // Skip file ignored warning: if a file is ignored by .eslintingore
+        // but linted explicitly (by arcanist), a warning will be reported,
+        // containing only: `{fatal:false,serverity:1,message:...}`.
+        if (strpos($offense['message'], "File ignored because") === 0) {
```

Comment:
This looks consistent with how ALE handles this case, too:

https://github.com/dense-analysis/ale/blob/c06467438dff38b939fd81b32395296d01a59df6/autoload/ale/handlers/eslint.vim#L196